crud/room_utils.py

```diff
@@ -0,0 +1,257 @@
+"""CRUD functions for Room, Facility, Feature and RoomType."""
```

Comment:
Added most of CRUD functions for Room, Facility, Feature and RoomType.
Possible filtering TBD